src/vue/modules/loyalty-points-statistics/components/viewers/withdrawals-viewer.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="withdrawals-viewer">
+    <line-chart
+      v-if="isLoaded"
+      id="withdrawals-chart"
+      :data="withdrawals"
+    />
+
+    <load-spinner
+      v-else-if="!isLoadFailed"
```

Comment:
`v-else-if="isLoadFailed"` and set here your `failed` message. Try not to use negation where it possible

Reply:
Fixed